src/SqlPersistence/Subscription/SqlSubscriptionFeature.cs

```diff
@@ -9,6 +9,7 @@ class SqlSubscriptionFeature : Feature
     SqlSubscriptionFeature()
     {
         DependsOn("NServiceBus.Features.MessageDrivenSubscriptions");
```

Comment:
This can be strongly typed in the new core 🥳

Reply:
Incorrect. It is still internal ;)